examples/AlwaysSucceeds.purs

```diff
@@ -37,26 +46,60 @@ main = launchAff_ $ do
     logInfo' "Running Examples.AlwaysSucceeds"
     validator <- liftContractM "Invalid script JSON" $ alwaysSucceedsScript
     vhash <- liftedM "Couldn't hash validator" $ validatorHash validator
+    logInfo' $ "Attempt to lock testada"
+    payToAlwaysSucceeds vhash validator
+    logInfo' $ "Try to spend locked values"
+    spendFromAlwaysSucceeds vhash validator
 
-    let
-      -- Note that CTL does not have explicit equivalents of Plutus'
-      -- `mustPayToTheScript` or `mustPayToOtherScript`, as we have no notion
-      -- of a "current" script. Thus, we have the single constraint
-      -- `mustPayToScript`, and all scripts must be explicitly provided to build
-      -- the transaction (see the value for `lookups` below as well)
-      constraints :: Constraints.TxConstraints Unit Unit
-      constraints = Constraints.mustPayToScript vhash unitDatum
-        $ Value.lovelaceValueOf
-        $ BigInt.fromInt 2_000_000
-
-      lookups :: Lookups.ScriptLookups PlutusData
-      lookups = Lookups.validator validator
-
-    ubTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
-    BalancedSignedTransaction bsTx <-
-      liftedM "Failed to balance/sign tx" $ balanceAndSignTx ubTx
-    txId <- submit bsTx.signedTxCbor
-    logInfo' $ "Tx ID: " <> show txId
+payToAlwaysSucceeds :: ValidatorHash -> Validator -> Contract () Unit
+payToAlwaysSucceeds vhash validator = do
+  let
+    -- Note that CTL does not have explicit equivalents of Plutus'
+    -- `mustPayToTheScript` or `mustPayToOtherScript`, as we have no notion
+    -- of a "current" script. Thus, we have the single constraint
+    -- `mustPayToScript`, and all scripts must be explicitly provided to build
+    -- the transaction (see the value for `lookups` below as well)
+    constraints :: Constraints.TxConstraints Unit Unit
+    constraints = Constraints.mustPayToScript vhash unitDatum
+      $ Value.lovelaceValueOf
+      $ BigInt.fromInt 2_000_000
+
+    lookups :: Lookups.ScriptLookups PlutusData
+    lookups = Lookups.validator validator
+
+  ubTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
+  BalancedSignedTransaction bsTx <-
+    liftedM "Failed to balance/sign tx" $ balanceAndSignTx ubTx
+  txId <- submit bsTx.signedTxCbor
+  logInfo' $ "Tx ID: " <> show txId
+
+spendFromAlwaysSucceeds :: ValidatorHash -> Validator -> Contract () Unit
+spendFromAlwaysSucceeds vhash validator = do
+  let
+    arbitraryRedeemer = unitRedeemer
+    scriptAddress = scriptHashAddress vhash
+  (UtxoM utxos) <-
+    fromMaybe (UtxoM Map.empty) <$> utxosAt scriptAddress
+  let
+    lookups :: Lookups.ScriptLookups PlutusData
+    lookups = Lookups.validator validator
+      <> Lookups.unspentOutputs utxos
+
+    orefs :: List Transaction.TransactionInput
+    orefs = fst <$> Map.toUnfoldable utxos
+
+    constraints :: TxConstraints Unit Unit
+    constraints =
+      fold
+        ( do
+            oref <- orefs
```

Comment:
You should try to spend just one unspent output at the script address I think. This will try to spend _all_ of them, no? There's likely a _lot_ locked at this address. This is the standard Plutus always succeeds script, it's not CTL-specific at all

Reply:
Yes, I was just translating the Plutus pioneer example for spending script outputs. I haven't looked that currently are around 14k founds locked by the script.